src/pages/index.tsx

```diff
@@ -29,6 +30,9 @@ const Home: NextPage = () => {
                 Welcome {me?.first_name} {me?.last_name}
               </h1>
               {JSON.stringify(me)}
+              <Link href={`/cmDashboard`}>
```

Comment:
OOC, why use backticks here, instead of say <Link href="/cmDashboard">? There is no variable substitution being done here.

Reply:
@amantri Thank you so much! I fixed it - I must have learned that from stack overflow, but now I have corrected the href to the proper format per the next.js docs.